intro-HPC/examples/Using-the-GPU-cluster-at-HPC-UGent/TensorFlow_GPU_joltik.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+#PBS -l walltime=1:0:0
+#PBS -l nodes=1:ppn=32,gpus=4
+
+module load PyTorch/1.2.0-fosscuda-2019.08-Python-3.7.2
```

Comment:
maybe the example should only use one of TF or pytorch?

Reply:
Yeah, that was meant as an example for you yo tweak.

I'll make it load just one module, but this isn't actually included in the documentation currently.

I can add it back though...